### PR TITLE
Update Parse.podspec with a later version of the FacebookSDK

### PR DESCRIPTION
**Pull request by adamlevy-emjay**
Updating the FacebookSDK to 11.2.1

### New Pull Request Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-iOS-OSX/security/policy).
- [ ] I am creating this PR in reference to an [issue](https://github.com/parse-community/Parse-SDK-iOS-OSX/issues?q=is%3Aissue).

### Issue Description

Related issue: #`FILL_THIS_OUT`

### TODOs before merging

- [ ] Add tests
- [ ] Add entry to changelog
- [ ] Add changes to documentation (guides, repository pages, in-code descriptions)

### CODE REVIEW

**Comment by mtrezza** [comment]
Kindly let me know when you're confident that it works and I'll start the CI

**Comment by adamlevy-emjay** [comment]
We are now using a forked version of the ParseSDK with the FacebookSDK updated.  Are you sufficient with just a "no issues found on our end?"

File: Parse.podspec [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  s.version          = '1.19.3'
+  s.version          = '1.19.4'
   s.license          =  { :type => 'BSD', :file => 'LICENSE' }
   s.homepage         = 'http://parseplatform.org/'
   s.summary          = 'A library that gives you access to the powerful Parse cloud platform from your iOS/OS X/watchOS/tvOS app.'
   s.authors          = 'Parse Community'
   s.social_media_url = 'https://twitter.com/ParsePlatform'
 
-  s.source           = { :git => 'https://github.com/parse-community/Parse-SDK-iOS-OSX.git', :tag => s.version.to_s }
+  s.source           = { :git => 'https://github.com/adamlevy-emjay/Parse-SDK-iOS-OSX.git', :tag => s.version.to_s }
```
Comment (mtrezza) [inline Parse.podspec:10]: This is pointing to your own fork, please change

**Review by mtrezza — CHANGES_REQUESTED** [review]

File: Parse.podspec (hunk 1 above)
Comment (adamlevy-emjay) [reply Parse.podspec:10]: Sorry let me fix it.

**Pull request closed without merging**